en/docs/administer/key-managers/configure-forgerock-connector.md

```diff
@@ -0,0 +1,288 @@
+# Configure Forgerock as a Key Manager
+
+WSO2 API Manager has the capability to support multiple Key Managers at the same time. So with the use of connectors, it is capable of supporting any authorization server as a Key Manager.
+Additionally it is prepacked with an inbuilt resident Key Manager.
+
+WSO2 API Manager can connect to Forgerock out-of-the-box using the [WSO2 API-M Forgerock Connector](https://github.com/wso2-extensions/apim-km-forgerock).
+
+Follow the instructions given below to configure Forgerock as a third-party Key Manager:
+
+### Configure Forgerock Server to work with APIM
+We assume that you already have a Forgerock installation and follow the steps below to configure the Fogerock Authorization server to work with WSO2 API Manager.
+
+#### Connfigure Oauth2 Provider
+1. Go to the URL where the Forgerock Authorization server is deployed and <br> navigate to 
+   Realms -> Top Level Realm. Top level realm which exists by default is the root realm and the examples and configurations in this guide are based on that.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-realms.png)
+
+2. Navigate to Root Realm -> Dashboard -> Configure Oauth provider -> Configure Ouath 2.0 and create an oauth provider by filling up the values.
+   
+    ![alt text]({{base_path}}/assets/img/administer/forgerock-createOauthProvider.png)   
```

Comment:
Do not use capital letters in file names as the image link will break when it is run on the live server.